playwright/e2e/crypto/utils.ts

```diff
@@ -139,12 +139,12 @@ export async function checkDeviceIsCrossSigned(app: ElementAppPage): Promise<voi
  * Check that the current device is connected to the expected key backup.
  * Also checks that the decryption key is known and cached locally.
  *
- * @param page - the page to check
+ * @param app - app page
  * @param expectedBackupVersion - the version of the backup we expect to be connected to.
  * @param checkBackupKeyInCache - whether to check that the backup key is cached locally.
```

Comment:
maybe you could clarify this while you are here:

```suggestion
 * @param checkBackupPrivateKeyInCache - whether to check that the backup decryption key is cached locally.
```